content/docs/jsx-in-depth.md

```diff
@@ -244,7 +244,7 @@ function App2() {
 }
 ```
 
-You can also pick specific props that your component will consume while passing all other props using the spread operator.
+You can also pick specific props that your component will consume while passing all other props using the spread syntax.
```

Comment:
There are other places with same error like 

>The array spread operator also lets you prepend an item by placing it *before* the original `...artists`:

Link to the line:
https://github.com/reactjs/reactjs.org/blob/b41b1dc35679c01c3252e7d512ce28c5e100d0a4/beta/src/pages/learn/updating-arrays-in-state.md?plain=1#L146

Lets fix other places too and make it consistent?

Reply:
thanks! I changed all "spread operator" to "spread syntax"